### DISCUSSION TITLE
Azuracast with Admin Panel?

### DISCUSSION THREAD

**Discussion by Hellfirehound** [AzuraCast/AzuraCast#5135] [Q&A]
Hi Folks,

I'm looking to host my website and my Azuracast station on the server.  Is this possible and, if so, what would I need to do to make sure it runs smoothly?

Am using an Ubuntu 20 VPS with Docker installed if that means anything to anyone...

**Comment by ItsMitchh** [comment]
If you want to run multiple sites on your server, then please follow our documentation here - https://docs.azuracast.com/en/administration/docker/multi-site-installation

**Reply by Hellfirehound** [reply]
Thanks for the info.

As a linux novice, this might go really well, or horribly wrong...

...stay tuned, so to speak
